tests/data/3GPP/3GPP_TSG_SA_WG2_UPCON.LOG1308A

```diff
@@ -0,0 +1,3639 @@
+=========================================================================
+Date:         Thu, 1 Aug 2013 10:46:56 +0000
+Reply-To:     Andreas Maeder <Andreas.Maeder@NECLAB.EU>
+From:         Andreas Maeder <Andreas.Maeder@NECLAB.EU>
+Subject:      UPCON conference calls preceding SA2#99
+Content-Type: multipart/alternative;
+              boundary="_000_3412F833DE9A814FA7E3E1B453E33E5F551D1053DAPHNISofficehd_"
+MIME-Version: 1.0
+
+--_000_3412F833DE9A814FA7E3E1B453E33E5F551D1053DAPHNISofficehd_
+Content-Type: text/plain; charset="iso-8859-1"
+Content-Transfer-Encoding: quoted-printable
+
+Dear all,
+
+
+I suggest to have two conference calls before SA2#99:
+
+1st CC: Sept. 5th, 15:00-17:00 (UTC+1)
+2nd CC: Sept. 12th, 15:00-17:00 (UTC+1)
+
+I propose the following rough plan for the 1st CC:
+-       Discuss objectives for UPCON work at SA2#99
+-       Presentation of contribution outlines/drafts to allow companies to =
+work on joint contributions
+
+I would expect the focus of UPCON work at SA2#99 as following:
+-       Resolution of open issues in already documented solutions
+-       No completely new solutions for Rel-12 time frame
+-       Discussion of evaluation criteria for solutions, and potentially of=
+ initial evaluations
+
+The coordinates will be distributed later.
+
+
+Have a nice vacation time, and kind regards
+   Andreas
+
+
+
+--_000_3412F833DE9A814FA7E3E1B453E33E5F551D1053DAPHNISofficehd_
+Content-Type: text/html; charset="iso-8859-1"
+Content-Transfer-Encoding: quoted-printable
+
+<html>
+<head>
+<meta http-equiv=3D"Content-Type" content=3D"text/html; charset=3Diso-8859-=
+1">
+<meta name=3D"Generator" content=3D"Microsoft Exchange Server">
+<!-- converted from rtf -->
+<style><!-- .EmailQuote { margin-left: 1pt; padding-left: 4pt; border-left:=
+ #800000 2px solid; } --></style>
+</head>
+<body>
+<font face=3D"Calibri" size=3D"2"><span style=3D"font-size:11pt;">
+<div>Dear all,</div>
+<div>&nbsp;</div>
+<div>&nbsp;</div>

```

Comment:
Is it legal for us to include this data? It looks like it is from 3GPP lists. Are there any licensing issues thare? @nllz